packages/lesswrong/server/repos/TagsRepo.ts

```diff
@@ -50,4 +50,61 @@ export default class TagsRepo extends AbstractRepo<DbTag> {
     const {count} = await this.getRawDb().one(`SELECT COUNT(*) FROM "Tags"`);
     return count;
   }
+
+  async getUsersMostFrequentlyCommentedTags(userId:string): Promise<DbTag[]> {
+    return this.any(`
+      SELECT
+        public."Tags".name,
+        COUNT(public."Comments"._id) AS comment_count
+      FROM public."Tags"
+      INNER JOIN public."TagRels" ON public."Tags"._id = public."TagRels"."tagId"
+      INNER JOIN
+          public."Comments"
+          ON public."TagRels"."postId" = public."Comments"."postId"
+      WHERE
+          public."Comments"."userId" = '${userId}'
```

Comment:
ayiiiiiiii!

Reply:
use query parameters, not string interpolation (we aren't using a library that lets us use template strings safely)